record_linux/lib/record_linux.dart

```diff
@@ -1,13 +1,12 @@
 import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
+import 'dart:ui';
```

Comment:
Is dart:ui import necessary?

Reply:
This is for VoidCallback. Will change to import 'package:flutter/foundation.dart'; which is more typical.